ONSdigital/rm-survey-service
# Added survey type column

Currently the system cannot differentiate between business, social and census surveys.

This PR adds a new `surveytype` column to the survey.survey table to be able to disinguish between different types of surveys.

**How to test:**

Run up the survey service and you should see the new column created and run `make test` to ensure the unit tests all pass.

This PR is linked to - https://trello.com/c/ZFF7x8Za
